release/go/go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/spacemeshos/api/release/go
 go 1.20
 
 require (
-	github.com/golang/protobuf v1.5.3
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.0
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20230822172742-b8732ec3820d
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20230920204549-e6e6cdab5c13
 	google.golang.org/grpc v1.58.2
 	google.golang.org/protobuf v1.31.0
 )
 
 require (
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
Where does it come from? I hoped that we could get rid of it finally :cry:

Reply:
The reason is that `google.golang.org/grpc` and `github.com/grpc-ecosystem/grpc-gateway/v2` depend on it. 

I assume to ensure compatibility between those libraries and `github.com/golang/protobuf` until the later is deprecated.

```
$ go mod graph | grep github.com/golang/protobuf@v1.5.3
...
github.com/grpc-ecosystem/grpc-gateway/v2@v2.18.0 github.com/golang/protobuf@v1.5.3
google.golang.org/genproto/googleapis/api@v0.0.0-20230920204549-e6e6cdab5c13 github.com/golang/protobuf@v1.5.3
google.golang.org/grpc@v1.58.2 github.com/golang/protobuf@v1.5.3
```